### ISSUE TITLE
missing gmp

### ISSUE THREAD

**Issue by rdp** [crystal-lang/homebrew-crystal#15]
No idea if this is the right place to report this, but at least with 0.18.2

{{{
$brew install crystal-lang
...
FH1002466:dev packrd$ crystal run rational.cr
ld: library not found for -lgmp
clang: error: linker command failed with exit code 1 (use -v to see invocation)
Error: execution of command failed with code: 1: `cc -o "/Users/packrd/.cache/crystal/crystal-run-rational.tmp" "${@}"  -rdynamic  -L/usr/lib -L/usr/local/lib -lgmp -lpcre -lgc -lpthread /usr/local/Cellar/crystal-lang/0.18.2/src/ext/libcrystal.a -levent -liconv -ldl`
FH1002466:dev packrd$ brew install gmp
==> Downloading https://homebrew.bintray.com/bottles/gmp-6.1.0.yosemite.bottle.tar.gz
Already downloaded: /Users/packrd/Library/Caches/Homebrew/gmp-6.1.0.yosemite.bottle.tar.gz
==> Pouring gmp-6.1.0.yosemite.bottle.tar.gz
🍺  /usr/local/Cellar/gmp/6.1.0: 17 files, 3.2M
FH1002466:dev packrd$ crystal run rational.cr
(success)
}}}

**Comment by asterite** [comment]
GMP is optional, if you don't use big numbers you don't need it. So we could maybe install it with homebrew, but I don't know. Eventually I think I'd like to implement big numbers without an extra library, just pure crystal.

**Comment by rdp** [comment]
yes I was using BigRational at the time.  However, the link line seems to
anticipate it, I'd add it to homebrew for now, since it's there and part of
the std. lib.
Cheers!
